pkg/nvml/zz_generated.api.go

```diff
@@ -0,0 +1,885 @@
+// Generated Code; DO NOT EDIT.
```

Comment:
Do we still want to add the license boilerplate?

Reply:
Added in latest